pennylane_lightning/_version.py

```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.31.0-dev10"
+__version__ = "0.31.0-dev11"
```

Comment:
Since this won't be hitting the 30.31 release, we can keep this in the 0.32 cycle